inventory/hosts

```diff
@@ -1,5 +1,5 @@
-[masters]
-gigabox ansible_host=10.0.0.5
+[control-plane]
```

Comment:
I would not use `-` in Ansible. There are templating horrors under the hood. You are lucky if it does not parse it incorrectly and tries to calculate control minus plane.

Reply:
Done https://github.com/roobre/fukuno/pull/2/commits/48bcea5a6909e1963337b20cface6c54b066b7a2